### PR TITLE
Create knownissues.adoc

### PR DESCRIPTION
**Pull request by jantley-ibm**

### CODE REVIEW

File: ref/general/reference/knownissues.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+:page-doc-category: Reference
+:linkattrs:
+:sectanchors:
+= Known issues
+
+Be aware of the following issues that can cause problems for your instance of Kabanero.
+
+== 
+
+There is a compatibility issue with the Appsody CLI versions 0.5.5 and 0.5.6 that can cause build pipelines to fail. These versions of the Appsody CLI append the image registry to the application configuration in the image tag. This metadata format can conflict with the format that is used in the Kabanero instance. If the applications image is active in the Kabanero instance, the pipeline continues, otherwise the build fails.
```
Comment (kilnerm) [inline ref/general/reference/knownissues.adoc]: Version 0.5.5 and 0.5.6 of the Appsody CLI will prepend the default image registry, docker.io, to the image tag in a new applications configuration regardless of whether the base collection specifies it. This results in an application configuration that conflicts with the meta-data used within the Kabanero instance that was created from the base collection configuration. This conflict causes the default build pipelines to fail for the application as it appears the relevant collection is not active within the Kabanero instance.

File: ref/general/reference/knownissues.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+After initializing the application, you can manually edit the `.appsody-config.yaml` file in the application source folder to remove the docker registry portion of the value of the image tag. Alternatively, use a different version of the Appsody CLI.
```
Comment (kilnerm) [inline ref/general/reference/knownissues.adoc]: Alternatively, upgrade your Appsody CLI to the latest version.

File: ref/general/reference/knownissues.adoc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+:page-layout: doc
+:page-doc-category: Reference
+:linkattrs:
+:sectanchors:
+= Known issues
+
+Be aware of the following issues that can cause problems for your instance of Kabanero.
+
+== 
+
```
Comment (kilnerm) [inline ref/general/reference/knownissues.adoc]: Can we have a title summarizing the problem, something like "Build pipelines fail when collection is active"

**Comment by jantley-ibm** [comment]
@kilnerm thanks for the feedback.  There are 3 PRs (#301 , #312 , and #314 ) for v0.2, 0.3, and 0.4, respectively.  All have been updated according to your changes.

**Pull request merged**